src/Proto.Persistence/Persistence.cs

```diff
@@ -28,7 +28,7 @@ public void Init(IProvider provider, IContext context)
             if (t != null)
             {
                 EventIndex = t.Item2;
-                Context.ReceiveAsync(t.Item1).Wait();
+                await Context.ReceiveAsync(t.Item1);

```

Comment:
Interesting.. there is the same: https://github.com/AsynkronIT/protoactor-dotnet/blob/master/src/Proto.Persistence/Persistence.cs#L48